### DISCUSSION TITLE
Add Deye SUN-XK-G05 String-Inverter to the list of supported Devices

### DISCUSSION THREAD

**Discussion by bastiitsab** [davidrapan/ha-solarman#328] [General]
Hi, I successfully tested the integration with a SUN-12K-G05 String-Inverter. I didn't found it in the [list of supported devices](https://github.com/davidrapan/ha-solarman/wiki/Supported-Devices#%EF%B8%8F-confirmed). Didn't found a way to directly add it.

**Comment by davidrapan** [comment]
Hi @bastiitsab, please give me link to deye web of your device (to product page if still sold unit or just pdf if it's the only option).

**Reply by bastiitsab** [reply]
Strange, cannot find it on the deye page. But here is the product sheet as pdf

[EN_SUN_4-12K-G05.pdf](https://github.com/user-attachments/files/18167041/EN_SUN_4-12K-G05.pdf)

**Reply by davidrapan** [reply]
Yes cause some devices are no longer obtainable directly through deye so only link to pdfs (for those devices) are available. 😉

_I added device to the list_.
